remake_depth.yml

```diff
@@ -0,0 +1,47 @@
+packages:
+  - yaml
+  - dplyr
+  - sbtools
+  - rgdal
+  - sp
+  - maptools
+  - rgeos
+  - magrittr
+  - plyr
+  - raster
+  - digest
+  - jsonlite
+
+sources:
+  - scripts/load_config.R
+  - scripts/link_to_nhd.R
+  - scripts/encryption_helpers.R
+  - scripts/depth_link_jacobson.R
+  - scripts/depth_summarize.R
+  - scripts/depth_link_lagos.R
+
+
+targets:
+  all: 
+    depends: 
+      - data/depth_data_linked/all_depths.edt
```

Comment:
does this need the key locally to work? is it a env.var or in the internal yaml?